unstructured/ingest/process.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import multiprocessing as mp
+from contextlib import suppress
+from functools import partial
+
+from unstructured.ingest.doc_processor.generalized import initialize, process_document
+from unstructured.ingest.interfaces import (
+    BaseConnector,
+    ProcessorConfigs,
+)
+from unstructured.ingest.logger import ingest_log_streaming_init, logger
+
+with suppress(RuntimeError):
+    mp.set_start_method("spawn")
+
+
+class Process:
```

Comment:
I suspect this was originally called MainProcess to not conflate with the individual pool processes. Any thoughts?

Reply:
Renamed the class to `Processor` since it's responsible for kicking off how ever many processes are allocated. Hopefully that makes it a bit clearer.